docs/source/models/intervention_models/alzheimers/testing_diagnosis.rst

```diff
@@ -30,4 +30,185 @@
+and simulants who satisfy all eligibility requirements at that time should be marked as having 
+previously recieved a CSF/PET test. These simulants will be ineligible for future 
+CSF/PET testing.
+
+BBBM testing
+~~~~~~~~~~~~
+
+BBBM testing is a hypothetical biomarker test which we will model in 
+:ref:`Alternative Scenario 1 <alz_scenarios>`. It will replace some CSF/PET testing and 
+assign positive/negative diagnosis which will inform treatment in :ref:`Alternative Scenario 2 <alz_scenarios>`.
+
+Year-specific testing rates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Testing rates do not vary by location, age or sex. 
+In 2020, 0% of eligible simulants are tested annually. This increases to 10% in 2030, 
+then increases linearly over time in each six-month period to reach 20% in 2035, to 40% in 2040 
+and then maxes out at 60% in 2045. 
+
+
+Implementation
+^^^^^^^^^^^^^^
+The simulant's baseline testing propensity will also be used as their BBBM testing propensity.
+
+On each timestep, use the following steps to assign CSF and PET tests:
+
+1. Assess eligibility based on the following requirements:
+
+  - Simulant is in pre-clinical stage
+  - Simulant age is >=60 and <80
+  - Simulant has not received a BBBM test in the last three years
+  - Simulant has never recieved a positive BBBM test
+  - Propensity is higher than year-specific testing rate
+
+2. If eligible (meets all requirements), give test. If not, do not give test.
+3. If given test, assign positive diagnosis to 90% of people and negative diagnosis to 10% of people. This 90% draw should be independent of any previous draws, eg people who test negative still have a 90% chance of being positive on a re-test.
+4. Record time of last test, yes/no diagnosis for future testing eligibility.
+
+Assumptions and Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Since BBBM testing eligibility is pre-clinical stage and CSF/PET is MCI stage, 
+  and simulants cannot move backwards, CSF/PET test history is irrelevant to BBBM 
+  test elibibility 
+- The same simulants undergo repeat testing to reflect ongoing issues with access or insurance,
+  so propensity does not need to be re-assigned at any point.
+- Since BBBM uses the same propensity as baseline testing, BBBM should mostly replace CSF and PET
+  testing, though some simulants may reach age 60 in the MCI stage, or get a BBBM false negative.
+
+.. todo::
+  People who are not simulated (will not develop AD dementia) will also be tested, and these tests,
+  including false positives, will need to be counted (outside the simulation).
+
+Initialization of test history
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+On initialization no one will have been tested. Due to test coverage jumping from 0% to 10% in 2030,
+we would expect a large group to be immediately tested and then a drop-off in testing counts.
+
+Observer
+~~~~~~~~
+Need an observer with test counts by location, age, sex, year, and diagnosis provided 
+
+Validation and Verification Criteria
```

Comment:
Similarly, this lives on the main concept model page here: https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_alzheimers/concept_model.html#v-v-tracking 

So you only need to do this when you make a run request.